jurity/_version.py

```diff
@@ -2,4 +2,4 @@
 # Copyright FMR LLC <opensource@fidelity.com>
 # SPDX-License-Identifier: Apache-2.0
 
-__version__ = "1.2.0"
+__version__ = "1.2.1"
```

Comment:
Can you update the version in the docs as well? https://github.com/fidelity/jurity/blob/master/docsrc/conf.py